manifests/r/RichWritesCorp/RWOS2Setup/1.0.0.0/RichWritesCorp.RWOS2Setup.installer.yaml

```
@@ -0,0 +1,15 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: RichWritesCorp.RWOS2Setup
+PackageVersion: 1.0.0.0
+InstallerSwitches:
+  Silent: 1
+Installers:
+- Architecture: neutral
+  InstallerType: exe
+  InstallerUrl: https://cdn.discordapp.com/attachments/794623099638775818/935271211523510342/RWOS2_Setup.exe

```

```suggestion
  InstallerUrl: https://cdn.discordapp.com/attachments/794623099638775818/935271211523510342/RWOS2_Setup.exe
```
It should come from an official domain; i.e. GitHub